paasta_tools/kubernetes_tools.py

```diff
@@ -1299,6 +1300,15 @@ def list_matching_deployments(
     )
 
 
+def replicasets_for_service_instance(
+    service: str, instance: str, kube_client: KubeClient
+) -> Sequence[V1ReplicaSet]:
+    return kube_client.deployments.list_namespaced_replica_set(
+        namespace="paasta",
+        label_selector=f"yelp.com/paasta_service={service},yelp.com/paasta_instance={instance}",
+    ).items
```

Comment:
Should this be wrapped in try catch, and return an error code? I guess it's OK to not catch the error as well. It's up to you. Looks good overall.

Reply:
Yeah, I want to keep the consistent between this and other functions, such as `list_matching_deployments`.